### PR TITLE
PReq for #1670: Make chain_parameters::current_fees const

### PR DESCRIPTION
**Pull request by nathanhourt**
#1670: Make const const again

### CODE REVIEW

File: libraries/egenesis/embed_genesis.cpp [hunk 1]
```diff
@@ -285,4 +285,4 @@ int main( int argc, char** argv )
    }
 
    return main_return;
-}
+} catch( fc::exception& e ) { edump((e)); throw; } }
```
Comment (pmconrad) [inline libraries/egenesis/embed_genesis.cpp]: `FC_LOG_AND_RETHROW()` ?

**Comment by pmconrad** [comment]
Looks good.

File: libraries/egenesis/embed_genesis.cpp (hunk 1 above)
Comment (nathanhourt) [reply libraries/egenesis/embed_genesis.cpp]: Eh, why not?

**Comment by pmconrad** [comment]
Doesn't compile with latest develop, it seems. Please check.

**Pull request merged**